micrometer-core/src/test/java/io/micrometer/core/instrument/binder/kafka/KafkaConsumerMetricsTest.java

```diff
@@ -35,44 +40,40 @@
 class KafkaConsumerMetricsTest {
     private final static String TOPIC = "my-example-topic";
     private final static String BOOTSTRAP_SERVERS = "localhost:9092";
+    private static int consumerCount = 0;
 
-    private final KafkaConsumerMetrics kafkaConsumerMetrics = new KafkaConsumerMetrics();
-
-    private static void createConsumer() {
-        Properties props = new Properties();
-        props.put(ConsumerConfig.BOOTSTRAP_SERVERS_CONFIG, BOOTSTRAP_SERVERS);
-        props.put(ConsumerConfig.GROUP_ID_CONFIG, "MicrometerTestConsumer");
-        props.put(ConsumerConfig.KEY_DESERIALIZER_CLASS_CONFIG, LongDeserializer.class.getName());
-        props.put(ConsumerConfig.VALUE_DESERIALIZER_CLASS_CONFIG, StringDeserializer.class.getName());
-
-        Consumer<Long, String> consumer = new KafkaConsumer<>(props);
-        consumer.subscribe(Collections.singletonList(TOPIC));
-    }
+    private List<Tag> tags = Arrays.asList(new ImmutableTag("app", "myApp"), new ImmutableTag("version", "1"));
```

Comment:
The right side can all be replaced with `Tags.of("app, "myapp", "version", "1");`

Reply:
Good point, addressed.